elementor/elementor
# Update conditions.php

[23-Dec-2024 10:25:16 UTC] PHP Warning:  Undefined array key "background_image" in /home/***/***/wp-content/plugins/elementor/includes/conditions.php on line 87
[23-Dec-2024 10:25:16 UTC] PHP Warning:  Trying to access array offset on value of type null in /home/***/***/wp-content/plugins/elementor/includes/conditions.php on line 90

## PR Checklist



## PR Type
What kind of change does this PR introduce?


## Summary
Testing the existence of an array element before value is assigned.

## Description
"PHP Warning:  Undefined array key "background_image" in /home/***/***/wp-content/plugins/elementor/includes/conditions.php on line 87
"PHP Warning:  Trying to access array offset on value of type null in /home/***/***/wp-content/plugins/elementor/includes/conditions.php on line 90

## Test instructions
This PR can be tested by following these steps:

Reload a page and check the error.log file for:
"PHP Warning:  Undefined array key "background_image" in /home/***/***/wp-content/plugins/elementor/includes/conditions.php on line 87
"PHP Warning:  Trying to access array offset on value of type null in /home/***/***/wp-content/plugins/elementor/includes/conditions.php on line 90

## Quality assurance


Fixes #
conditions.php on line 87
conditions.php on line 90
